test/Blazor.Extensions.Logging.Test/wwwroot/index.html

```diff
@@ -1,13 +1,17 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width">
-    <title>Blazor.Extensions.Logging.Test</title>
-    <base href="/" />
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width">
+  <title>Blazor.Extensions.Logging.Test</title>
+  <base href="/" />
+  <script src="_content/blazor.extensions.logging.js"></script>
```

Comment:
At least in my case, this needs to be `_content/blazor.extensions.logging/blazor.extensions.logging.js`.

(You can find the correct path by looking in `bin\…\YourProjectName.StaticWebAssets.xml`.)

Reply:
when this PR will be completed and publish to nuget? current version doesnt work with netstandard2.1

Reply:
bump, can we get this pushed to nuget?